Dockerfile.venv

```diff
@@ -0,0 +1,21 @@
+ARG python_version
+FROM elifesciences/python:${python_version}
+
+ARG pipenv_dev_arg
+
+ENV LANG=C.UTF-8
```

Comment:
Not needed when https://github.com/elifesciences/elife-base-images/pull/20 is merged.